en/qgep_swmm/Check-SWMM-File.rst

```diff
@@ -93,3 +93,22 @@ Missing population densities
 ``qgep_od.catchment_area.population_density_current`` or ``qgep_od.catchment_area.population_density_planned`` are not filled.
 
 
+Subcatchment and rain Gages
```

Comment:
Subcatchment and Rain gauges

Reply:
I think it is rain gauges - not rain gages - see https://en.wikipedia.org/wiki/Rain_gauge